firefox-ios/Client/Frontend/Fakespot/Views/CircularProgressView.swift

```diff
@@ -6,8 +6,9 @@ import SwiftUI
 import Common
 
 struct CircularProgressView: View, ThemeApplicable {
-    @Environment(\.themeType)
-    var themeVal
+    let windowUUID: WindowUUID
```

Comment:
I think this is already defined in `FakespotMessageCardViewModel`

Reply:
Thanks for catching 👍, I've added a ToDo, will clean this up in the next follow-up PR for 8313